### PR TITLE
Added getTotalFatSamples to the library.

### PR DESCRIPTION
**Pull request by bvelasquez**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Added getTotalFatSamples to supplement existing getCarbohydrateSamples and getProteinSamples. This is needed in order to pull all relevant health "macros" for dietary apps.

Fixes # (issue)

https://github.com/agencyenterprise/react-native-health/issues/107

## Type of change

Please delete options that are not relevant.

- [x ] New feature (non-breaking change which adds functionality)
- [x ] This change requires a documentation update

## Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by Macelai** [comment]
hey @bvelasquez, thanks for your contribution, amazing job!

Can you please add a documentation for this method as we have for the others:

**Review by Macelai — CHANGES_REQUESTED** [review]
Please add documentation

**Comment by bvelasquez** [comment]
Added documentation @macelai .

File: index.d.ts [hunk 1]
```diff
@@ -445,15 +450,6 @@ declare module 'react-native-health' {
     end: string
   }
 
-  export interface ElectrocardiogramSampleValue extends BaseValue {
```
Comment (macelai) [inline index.d.ts:448]: Hey, I think this should not be deleted

**Review by macelai — CHANGES_REQUESTED** [review]
Thanks for the documentation, only this part is kinda weird

File: index.d.ts (hunk 1 above)
Comment (bvelasquez) [reply index.d.ts:448]: @macelai very weird. I've merged master to my fork. It's showing "up to date" and no changes, but these lines are missing from my fork and not merging from master. I can add them back manually.

Comment (bvelasquez) [reply index.d.ts]: @macelai Added it back.

**Comment by bvelasquez** [comment]
@macelai Let me know if anything else. thanks!

**Comment by macelai** [comment]
Great job!

**Review by macelai — APPROVED** [review]

**Pull request merged**